lib/logstash/plugin_mixins/elasticsearch/api_configs.rb

```diff
@@ -66,6 +66,8 @@ module APIConfigs
         # Set the keystore password
         :keystore_password => { :validate => :password },
 
+        :ssl_supported_protocols => { :validate => ['TLSv1.1', 'TLSv1.2', 'TLSv1.3'], :default => [], :list => true },
```

Comment:
[ES](https://www.elastic.co/guide/en/elasticsearch/reference/current/security-settings.html) also (still) supports: `SSLv3`, `TLSv1` and `SSLv2Hello` but since these are disabled by default in an up-to-date Java I do not think we need to support them give even an old 6.0 cluster is expected to run on at least Java 8.

Reply:
I'm slightly wary of removing something that is a valid option with Elasticsearch, maybe we should keep it but warn?

Reply:
we had a discussion with @jsvd on the topic and part of that outcome was that we start with `>= TLSv1.1` (for all plugins), in this case despite ES' support for legacy `SSLv3` and `TLSv1`.
existing plugin versions using TLS <= 1.1, on an up-to-date JDK, need tampering with the java.security files so if they're upgrading they either upgrade to secure TLS or we'll hear them complaining on the feature per plugin basis (the default - not setting the option is expected to work for them as before).